exercises/rock_paper_scissors.livemd

```diff
@@ -107,12 +114,34 @@ Bind a `player1_choice` and `player2_choice` variable to `:rock`, `:paper`, or `
 Enter your solution below.
 
 ```elixir
-
+player1_choice = Enum.random([:rock, :paper, :scissors])
+IO.inspect(player1_choice, label: "Player1")
+
+player2_choice = Enum.random([:rock, :paper, :scissors])
+IO.inspect(player2_choice, label: "Player2")
+
+# case {player1_choice, player2_choice} do
+#   {_val, _val} -> "Draw"
+#   {:rock, :paper} -> "Player 2 wins!"
+#   {:rock, :scissors} -> "Player 1 wins!"
+#   {:paper, :rock} -> "Player 1 wins!"
+#   {:paper, :scissors} -> "Player 2 wins!"
+#   {:scissors, :rock} -> "Player 2 wins!"
+#   {:scissors, :paper} -> "Player 1 wins!"
+# end
+
+case {player1_choice, player2_choice} do
+  {_val, _val} -> "Draw"
```

Comment:
clever! this way you only need to handle one case.

One consideration for this style of logic is that if the player choices are invalid, then player 2 would always win.

That might not be a concern if you can guarantee valid input, but it's something to consider.

Reply:
Ah, Noted! Didn't think of that. Thanks